app/models/miq_template.rb

```diff
@@ -32,6 +32,10 @@ def self.eligible_for_provisioning
     where(:type => subclasses_supporting(:provisioning).map(&:name)).active
   end
 
+  def self.search(name)
+    where('vms.name like ?', "%#{name}%")
```

Comment:
this breaks when we search for something like - `cent%`
```
Started POST "/vm_cloud/vm_pre_prov/new?search_text=cent%&hide_deprecated_templates=true" for ::1 at 2024-04-17 19:40:06 +0530
[----] F, [2024-04-17T19:40:06.866328 #13781:ece0] FATAL -- :   
ActionController::BadRequest (Invalid query parameters: invalid %-encoding (cent%)):
```

Reply:
Hey @Fryguy , I had made a small change here. 
- If we search for `cent%`, it's going to search for `cent\\%`
- If we search for `cent%_`, it's going to search for `cent\\%\\_`

if we search for `cent`, it would return - **10** records, but would return **0**, it we search for `cent%`